src/sql/sim.sql

```diff
@@ -0,0 +1,33 @@
+CREATE TABLE sim(
+  "iter" INTEGER,
+  "token" TEXT,
+  "all_similarity" REAL,
+  "is_hold_out" REAL,
```

Comment:
not bool?

Reply:
yeah that should be a bool

Reply:
actually going to be text. sqlite does not have booleans